.github/workflows/phpcs.yml

```diff
@@ -1,25 +1,13 @@
 name: PHP Coding Standards
+
 on:
   pull_request:
-  workflow_dispatch:
+
 jobs:
   phpcs:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.11.0
-        with:
-          all_but_latest: true
-      - name: Checkout code
-        uses: actions/checkout@v3
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: '8.0'
-          coverage: none
-      - name: Validate Composer package
-        run: composer validate --strict
-      - name: Install dependencies
-        uses: ramsey/composer-install@v2
-      - name: Run PHPCS
-        run: composer phpcs
+    strategy:
+      matrix:
+        php: [ 8.0 ]
+    uses: alleyinteractive/.github/.github/workflows/php-coding-standards.yml@main
```

Comment:
Suggestion: https://github.com/alleyinteractive/.github?tab=readme-ov-file#php-composer-script

To use both `phpcs` and `phpunit` in a single action.